apps/minifront/src/components/v2/dashboard-layout/index.tsx

```diff
@@ -30,16 +30,14 @@ export const DashboardLayout = () => {
 
       <Grid tablet={8} desktop={6} xl={4}>
         <Card title={CARD_TITLE_BY_PATH[v2PathPrefix(pagePath)]}>
-          <div className='flex flex-col gap-4'>
```

Comment:
This wrapper is no longer needed, since `<Card />` now applies a spacing of `4` to all its direct children